config/mutant.yml

```diff
@@ -0,0 +1,26 @@
+---
+integration:
+  name: rspec
+requires:
+  - memoizable
+matcher:
+  subjects:
+    - Memoizable*
+  ignore:
+    - Memoizable::InstanceMethods#freeze
+    - Memoizable::Memory#[]
+    - Memoizable::Memory#[]=
+    - Memoizable::Memory#fetch
+    - Memoizable::Memory#initialize
+    - Memoizable::Memory#key?
+    - Memoizable::Memory#marshal_load
+    - Memoizable::MethodBuilder#call
+    - Memoizable::MethodBuilder#create_memoized_method
+    - Memoizable::MethodBuilder#initialize
+    - Memoizable::MethodBuilder#remove_original_method
+    - Memoizable::MethodBuilder#set_method_visibility
+    - Memoizable::ModuleMethods#included
+mutation:
+  timeout: 1.0
+coverage_criteria:
```

Comment:
@dkubb AFAIK this code does not have any loops where mutant can mutate to an infinite loop, for this reason I'd rather have timeouts reported as alive mutations which would make you bump the timeout.

Reply:
@mbj you think I should remove the timeout option? Or should I decrease it to something like 0.1 or lower?

Reply:
@dkubb I think its not bad to have the timeout option, I know some cases where rspec hangs up in rendering exceptions (bad source location algo), it would at least make sure mutant terminates showing you the cases rspec hangs.

Reply:
@dkubb Bump, as mutant should not result in mutations to infinite loops here (there are no loops with custom control in this code base) no mutation should ever result in an infinite loop, and thus coutning timeouts that arise from infinite loops as a kill is IMO redundant.